### DISCUSSION TITLE
Cooling Mode (Kühlbetrieb HK1)

### DISCUSSION THREAD

**Discussion by Karssi** [pail23/stiebel_eltron_isg_component#285] [Q&A]
Hello.

I have a LWZ404 with an ISG+ and have been using this integration for a while now and it works very well.
Now in the summer I would like to be able to turn ON/OFF the Cooling Mode (Kühlbetrieb) via Home Assistant.
When it is hot outside I usually turn it on during the day (when the solar panels provide the all the necessary power for it) and off in the evening. If I forget to do this it will consume so much energy during the night that it will empty out my 22kWh battery by about 3 in the morning.
So an automation via HA would be welcome.
I can control it via the ISG Web Interface but I can't find the option in the Home Assistant Integration.

Is there a way to add this feature? Or is somewhere hidden or do I need a ISG software update?

Many Thanks!
Michiel

**Comment by xylone** [comment]
I think you mean this option in the ISG webinterface:
<img width="979" alt="Screenshot 2024-09-10 at 14 07 14" src="https://github.com/user-attachments/assets/cff58eec-9181-4bad-af08-c2ad6d554d1a">

I also would like to be able to control this, but for the opposite purpose. I want to turn it off when the room is cold enough for us. The WPF COOL 07 just keeps cooling until 20/21 degrees and that is way too cold.

But I'm almost certain this register is not exposed by the ISG...  Which should of course.

**Reply by xylone** [reply]
Maybe @mbeckerkassel knows which register this is?

**Reply by ChristophWeigert** [reply]
Also interested in this option. Either set the on/off or modify the required outdoor temperatur after which the cooling is started.

**Comment by z3us22** [comment]
I just looked for this page - I would be interested as well in this feature since the "cooling time setting" (which you can set up in the Stiebel Eltron setting) for the cooling is not working

**Comment by Optic00** [comment]
Cooling mode control is tracked centrally in issue #413. The register reported there for THZ 504 is still unverified with respect to address convention, function code, and write semantics, so that issue remains open. Closing this discussion in favor of #413.
